general/error.hpp

```diff
@@ -144,4 +144,10 @@ void mfem_warning(const char *msg = NULL);
    "invalid index " #i << " = " << (i) << \
    ", valid range is [" << (imin) << ',' << (imax) << ')')
 
+#if defined(__CUDA_ARCH__)
+#define MFEM_ABORT_KERNEL(msg) asm("trap;");
```

Comment:
Remove the last `;` from here -- it is better to have the `;` when using the macro: `MFEM_ABORT_KERNEL("message");`.